docs/contributing/frontend.md

```diff
@@ -0,0 +1,133 @@
+# Frontend
+
+This is a guide to help the Coder community and also Coder members contribute to our UI. It is ongoing work but we hope it provides some useful information to get started. If you have any questions or need help, please send us a message on our [Discord server](https://discord.com/invite/coder). We'll be happy to help you.
```

Comment:
Is there any place where we can find a cheatsheet with commands? vite, yarn, how to start Coder in development mode, etc.

Reply:
Nops, but it is a good idea! I'm going to send this in a next PR.